src/DBFTPlugin/DBFTPlugin.cs

```diff
@@ -53,10 +53,11 @@ private void OnStart()
             Start(walletProvider.GetWallet());
         }
 
-        public void Start(Wallet wallet)
+        public void Start(Wallet wallet, Settings settings = null)
         {
             if (started) return;
             started = true;
+            if (settings != null) this.settings = settings;
             consensus = neoSystem.ActorSystem.ActorOf(ConsensusService.Props(neoSystem, settings, wallet));
```

Comment:
you're missing a `this` on line 60. when there's a method parameter and class field with the same identitifer (aka `settings`) you need to use `this.` to distinguish between them.  The way the code is now, the settings passed to `ConsensusService.Props` will always be the parameter `settings`, even if it's null.

``` csharp
consensus = neoSystem.ActorSystem.ActorOf(ConsensusService.Props(neoSystem, this.settings, wallet));
```